SPEC.md

```diff
@@ -2997,13 +2997,17 @@ A WDL task can be thought of as a template for running a set of commands - speci
 
 A task is defined using the `task` keyword, followed by a task name that is unique within its WDL document.
 
-A task has a required [`command`](#command-section) that is a template for a Bash script.
+Tasks are comprised of the following elements:
 
-Tasks explicitly define their [`input`s](#task-inputs) and [`output`s](#task-outputs), which is essential for building dependencies between tasks and workflows. The value of an input declaration may be supplied by the caller. Tasks may have additional "private" declarations within the task body. All task declarations may be initialized with hard-coded literal values, or may have their values constructed from expressions. Input and private declarations can be referenced in the command template.
+* A singular, optional [`input`](#task-inputs) section, which defines the inputs for the task.
+* A singular, required [`command`](#command-section), which defines the Bash script to be executed.
+* A singular, optional [`output`](#task-outputs) section, which defines the outputs for the task.
+* A singular, optional [`runtime`](#runtime-section) section, which defines the required runtime environment conditions.
```

Comment:
We are deprecating `runtime` in favor of `requirements` and `hints` but that PR (#595) isn't merged yet. I think it makes sense to merge this PR first and then updated the above in #595.

Reply:
I have updated this and added a 🗑️ to the `runtime` section so it can be removed at the same time as the section itself.